ExoPlayerAdapter/src/main/java/com/mux/stats/sdk/muxstats/internal/BandwidthMetric.kt

```diff
@@ -285,26 +278,23 @@ internal class BandwidthMetricDispatcher(player: ExoPlayer,
     fun onLoadCompleted(
         loadTaskId:Long, segmentUrl:String?, bytesLoaded:Long, trackFormat:Format?,
         responseHeaders:Map<String, List<String>>) {
-        if (player == null  || collector == null
-            || currentBandwidthMetric() == null) {
```

Comment:
Is currentBandwidthMetric now guaranteed to not be null?

Reply:
Yes. You can tell at a glance because it's type is `BandwidthMetricHls` and not `BandwidthMetricHls?`.